docs/modules/ROOT/pages/api/erc721.adoc

```diff
@@ -506,7 +506,7 @@ Requirements:
 
 [.contract-item]
 [[ERC721-_safe_mint]]
-==== `[.contract-item-name]#++_safe_mint++#++(ref self: ContractState, to: ContractAddress, token_id: u256, data: Span<felt252>)++` [.item-kind]#internal#
+==== `[.contract-item-name]#++_safe_mint++#++(ref self: ContractState, to: ContractAddress, token_id: u256, data: ByteArray)++` [.item-kind]#internal#
```

Comment:
data makes more sense as a `Span<felt252>`, because this is "Additional data with no specified format" that could be serialized into potentially anything. In Solidity is passed as bytes, but here in Cairo a type is passed down as Span<felt252> for Serde.

Reply:
This would apply to the rest of the PR as well.

Reply:
Yes, agreed